R/editSVG.R

```diff
@@ -0,0 +1,21 @@
+#' Run plotting code and open svg in OS/system default svg viewer or editor.
+#'
+#' This is useful primarily for testing or post-processing the SVG.
+#'
+#' @param code Plotting code to execute.
+#' @param ... Other arguments passed on to \code{\link{devSVG}}.
+#' @export
+#' @examples
+#' editSVG(plot(1:10))
+#' editSVG(hist(rnorm(100)))
+
+editSVG <- function(code, ...) {
+  tmp <- tempfile( fileext = ".svg" )
+  devSVG(tmp, width = par("din")[1], height = par("din")[2], ...)
```

Comment:
probably right, but I wondered if a file name argument might make sense in the `editSVG` function.  I think I would almost always start without a file name and then save as from Illustrator/Inkscape, but some might prefer to create a named file directly.